Source/ZoomNet.IntegrationTests/Tests/Reports.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using ZoomNet.Models;
+using ZoomNet.Utilities;
+
+namespace ZoomNet.IntegrationTests.Tests
+{
+	public class Reports : IIntegrationTest
+	{
+		public async Task RunAsync(string userId, IZoomClient client, TextWriter log, CancellationToken cancellationToken)
+		{
+			if (cancellationToken.IsCancellationRequested) return;
+
+			await log.WriteLineAsync("\n***** REPORTS *****\n").ConfigureAwait(false);
+
+			//GET ALL MEETINGS
+			var totalMeetings = await client
+				.Meetings.GetAllAsync(userId, MeetingListType.Scheduled, 30, 1, cancellationToken)
+				.ConfigureAwait(false);
+
+			var scheduledMeetings = totalMeetings.Records.OfType<ScheduledMeeting>()
+				.Where(x => x.StartTime.AddMinutes(x.Duration) < DateTime.UtcNow.AddDays(-1));
+
+			int totalParticipants = 0;
+
+			foreach (var meeting in scheduledMeetings)
+			{
+				try
+				{
+					var paginatedParticipants = await client.Reports.GetMeetingParticipantsAsync(meeting.Id.ToString(), null, 30, null, cancellationToken);
```

Comment:
Another guess on my part: maybe the API expects the meeting UUID rather than the Id? Maybe that's the root cause of the "meeting doesn't exist" problem you are experiencing?

Reply:
I changed the test to pull past instances of the meetings and used the UUID instead.  That fixed it except in instances where the UUID starts with / (even with double encoding on my end, but I need to do some testing to see what I am doing wrong).  I ignored that use case in my test so that it was able to successfully run everywhere I tested.